config/alfajores-celo-testnet.json

```diff
@@ -0,0 +1,19 @@
+{
+	"network": "alfajores-celo-testnet",
```

Comment:
@swetshaw added this network to the graph node?

Reply:
Yes @adiwajshing , creating a PR in that repo